ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -873,6 +873,9 @@ public class BottomSheetController: UIViewController {
 
     private let shouldShowDimmingView: Bool
 
+    private lazy var shadowColor = UIColor(colorValue: self.view.fluentTheme.globalTokens.neutralColors[.black]).cgColor
```

Comment:
if we are going to define shadow colors this way, why do we even have shadow tokens? https://github.com/microsoft/fluentui-apple/blob/fluent2-colors/ios/FluentUI/Core/Theme/Tokens/AliasTokens.swift#:~:text=public-,enum,-ShadowColorsTokens%3A%20CaseIterable

Reply:
FWIW this isn't a tokenized control yet, right?

Reply:
talked offline with @mischreiber but even if it isn't tokenized control we should be using whatever set of defined shadow token color. If not, why even have those alias tokens.